lib/drivers/insuletDriver.js

```diff
@@ -508,6 +508,8 @@ module.exports = function (config) {
       var prog = struct.unpack(basalProgramsHeader.rawdata, prgOffset,
                                fixedRecords.basal_programs_name.format,
                                fixedRecords.basal_programs_name.fields);
+      prog.name = prog.name.replace(/\./g, '-');
```

Comment:
I guess the thing that worries me with the drivers is that lack of tests around them. In this instance maybe we could have a func that is used but can also be independently tested without taking the rest of the drive into consideration. We know that if the name comes in as this format we need to update it to be that format otherwise bad things can happen.

Thoughts?